pkg/ibm_plugin/sdk/sdk_test.go

```diff
@@ -21,9 +21,26 @@ package ibm
 import (
 	"testing"
 
+	ibmCommon "github.com/paraglider-project/paraglider/pkg/ibm_plugin"
 	"github.com/stretchr/testify/require"
 )
 
+var resourceGroupID = ibmCommon.GetIBMResourceGroupID()
```

Comment:
This isn't used and is causing the test to fail since the env variable isn't set. The unit tests should be able to run regardless of whether the cloud creds, etc are set up

Reply:
Hey @smcclure20,
- `resourceGroupID` is used in the cleanup function `TestCleanup`. IDE might have failed to pick it up due to the build tag.
- `TestCleanup` was removed to avoid needlessly invoking it via the `make test` workflow.
- Unit tests were decoupled from the `PARAGLIDER_IBM_RESOURCE_GROUP_ID` environment var.